opcUaDevSupApp/Makefile

```diff
@@ -27,8 +27,13 @@ USR_INCLUDES += -I$(UASDK)/include/uaclient
 USR_INCLUDES += -I$(UASDK)/include/uapki
 USR_INCLUDES += -I$(UASDK)/include/uastack
 USR_INCLUDES += -I$(UASDK)/include/xmlparser
+USR_INCLUDES += -I$(BOOST)
 
-EXTLIB_INSTALLS += $(INSTALL_LIB)/libuastack.so
+ifeq ($(OS), Windows_NT)
+    EXTLIB_INSTALLS += $(INSTALL_LIB)/uastack.lib
```

Comment:
I simplified the Makefile a bit. Unfortunately I don't know how to do it in a better way in order to solve issue #3.